app/controllers/webpages_controller.rb

```diff
@@ -2,6 +2,7 @@
 
 class WebpagesController < ApplicationController
   before_action :set_webpage, only: :show
+  before_action :enable_maximum_privacy_headers, only: :search
```

Comment:
@cdoyle-temple I'm wondering whether this also needs to be reverted and happen at ingress.